code/game/objects/items.dm

```diff
@@ -129,14 +129,28 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 	var/heat = 0
 	/// IS_BLUNT | IS_SHARP | IS_SHARP_ACCURATE Used to define whether the item is sharp or blunt. IS_SHARP is used if the item is supposed to be able to cut open things. See _DEFINES/combat.dm
 	var/sharpness = IS_BLUNT
+	//this multiplies an attacks force for secondary effects like attacking blocking implements, dismemberment, and knocking a target silly
+	var/attack_weight = 1
 
 	/// What this thing does when used like a tool. NONE if it isn't a tool. If I give a piece of paper TOOL_WRENCH I can use it to unwrench tables. See _DEFINES/tools.dm
 	var/tool_behaviour = NONE
 	/// The tool speed multiplier of how long it takes to do the tool action.
 	var/toolspeed = 1
 
 	/// The chance that holding this item will block attacks.
-	var/block_chance = 0
+	var/block_level = 0
```

Comment:
Couldn't 4 of these be bitflags. I'd like to reduce the # of vars on item as much as possible

Reply:
done